### PR TITLE
Revert "Hydra search path correction"

### PR DESCRIPTION
**Pull request by ChataingT**
Reverts ChataingT/sam2_longer_video#2

Sorry, should have tested it before.

The new hydra path does not pass the test.
The reason is hydra need a __init__ in the direcotry of config.

Has it was implemented, the parameter given for the config should be "configs/sam2.1/sam2.1_hiera_[size].yaml"

It pas the local test inside the docker

### CODE REVIEW

**Comment by ChataingT** [comment]
Can you check that your error was because of the path used ?

**Comment by Nico-La-Porta** [comment]
I'll check the compatibility with the local test inside the docker as soon as possible.
I tried it in SUPSI's HPC and it worked properly (test_sam_manager passed)

**Comment by Nico-La-Porta** [comment]
Solved.

Directories containing a dot (".") in their name messed up the hydra config search path

Now it passes the test

File: sam2/__init__.py [hunk 1]
```diff
@@ -8,4 +8,4 @@
 from hydra.core.global_hydra import GlobalHydra
 
 if not GlobalHydra.instance().is_initialized():
-    initialize_config_module("sam2/configs/sam2.1", version_base="1.2")  # Modify the path to the config files folder
```
Comment (Nico-La-Porta) [inline sam2/__init__.py:11]: The error raised due to the dot in "sam2/configs/sam2.1", misinterpreting the correct path

File: test/test.py [hunk 2]
```diff
@@ -44,14 +44,14 @@ def test_one_mask_bedroom(self):
 
         this_file_path = pathlib.Path(__file__).parent.resolve()
         sam2_checkpoint = os.path.join(this_file_path,"../checkpoints/sam2.1_hiera_tiny.pt")
-        model_cfg = "configs/sam2.1/sam2.1_hiera_t.yaml"
```
Comment (Nico-La-Porta) [inline test/test.py:47]: Once the hydra search path is correct, the import becomes relative to that path

File: test/test.py [hunk 3]
```diff
@@ -44,14 +44,14 @@ def test_one_mask_bedroom(self):
...
+        model_cfg = "./sam2.1_hiera_t.yaml"
 
         predictor = build_sam2_video_predictor(model_cfg, sam2_checkpoint, device=device)
 
         # `video_dir` a directory of JPEG frames with filenames like `<frame_index>.jpg`
         video = os.path.join(this_file_path,"./assets/bedroom.mp4")
 
-        inference_state = predictor.init_state(video_path=video
```
Comment (Nico-La-Porta) [inline test/test.py:54]: missing comma

**Pull request merged**